docs/docs/workspaces/README.md

```diff
@@ -0,0 +1,93 @@
+# Workspaces
```

Comment:
Is this content a duplicate of `workspace-settings/README.md`?

Reply:
Yup structural changes finalized 👍